akka-distributed-data/src/main/resources/reference.conf

```diff
@@ -65,7 +65,7 @@ akka.cluster.distributed-data {
     # Some complex deltas grow in size for each update and above this
     # threshold such deltas are discarded and sent as full state instead.
     # This is number of elements or similar size hint, not size in bytes.
-    max-delta-size = 200
+    max-delta-size = 50
```

Comment:
The ratio is also different where if you divide the former by 2 and wanted to divide the later by 2 as well, `max-delta-size` should then be 100 vs 50. 
I'm just wondering if that is the intention?
And if so, how the numbers were arrived at?

Reply:
Those two properties are not related, but could result in similar problems. New values are taken still a only a guess (based on slightly better knowledge now with some insights from production system). In the end it is application specific and depends on the data sizes of the entries.

`max-delta-elements` is used for the full gossip, e.g. when syncing up a new node joining the cluster. Default payload size in Artery is 256kB so this new value makes room for 500B for each top level DData entry.

`max-delta-size` is used when aggregating CRDT delta updates (batching before sending). With frequent updates to the same entry the previous value caused the deltas to grow too big.